src/jsx/import/session-type/data-append.jsx

```diff
@@ -169,65 +168,66 @@ var DataAppend = React.createClass({
+            data: null,
         };
 
-      // boolean to show ajax spinner
-        this.setState({display_spinner: true});
-
-      // asynchronous callback: ajax 'done' promise
-        ajaxCaller(function (asynchObject) {
-          // Append to DOM
-            if (asynchObject && asynchObject.error) {
-                this.setState({ajax_done_error: asynchObject.error});
-            } else if (asynchObject) {
-                this.setState({ajax_done_options: asynchObject});
-            }
-        // boolean to hide ajax spinner
-            this.setState({display_spinner: false});
-        }.bind(this),
-      // asynchronous callback: ajax 'fail' promise
-        function (asynchStatus, asynchError) {
-            if (asynchStatus) {
-                this.setState({ajax_fail_status: asynchStatus});
-                console.log('Error Status: ' + asynchStatus);
-            }
-            if (asynchError) {
-                this.setState({ajax_fail_error: asynchError});
-                console.log('Error Thrown: ' + asynchError);
-            }
-        // boolean to hide ajax spinner
-            this.setState({display_spinner: false});
-        }.bind(this),
-      // pass ajax arguments
-        ajaxArguments);
-    },
-    componentWillMount: function() {
-      // update redux store
-        const actionLayout = setLayout({'layout': 'analysis'});
+        // boolean to show ajax spinner
+        this.setState({ display_spinner: true });
+
+        // asynchronous callback: ajax 'done' promise
+        ajaxCaller(
+            (asynchObject) => {
+                // Append to DOM
+                if (asynchObject && asynchObject.error) {
+                    this.setState({ ajax_done_error: asynchObject.error });
+                } else if (asynchObject) {
+                    this.setState({ ajax_done_options: asynchObject });
+                }
+                // boolean to hide ajax spinner
+                this.setState({ display_spinner: false });
+            },
+            // asynchronous callback: ajax 'fail' promise
+            (asynchStatus, asynchError) => {
+                if (asynchStatus) {
+                    this.setState({ ajax_fail_status: asynchStatus });
+                    console.log(`Error Status: ${asynchStatus}`);
+                }
+                if (asynchError) {
+                    this.setState({ ajax_fail_error: asynchError });
+                    console.log(`Error Thrown: ${asynchError}`);
+                }
+                // boolean to hide ajax spinner
+                this.setState({ display_spinner: false });
+            },
+            // pass ajax arguments
+            ajaxArguments,
+        );
+    }
+    componentWillMount() {
+        // update redux store
+        const actionLayout = setLayout({ layout: 'analysis' });
         this.props.dispatchLayout(actionLayout);
 
-        const actionContentType = setContentType({'layout': 'data_append'});
```

Comment:
I wasn't very good at being consistent with the quotes.  Some states had them, others didn't. Thanks for catching this.